Review and update README

lread opened the pull request:
As a noob, I did not at first understand Polylith was separate from the Clojure poly tool and documented separately. I made this clearer in the README.
- changed the title to include "for Clojure"
- I used the term "conceptual" instead of "high-level" for Polylith gitbook docs.

I got very confused by the diagram.
It is very cool but I thought it deserved at least a high level description. I might have got it right, please do review.

Was not shy about using the word Polylith.
In a README, I felt it deserved to be repeated often.

Other changes were minor, like splitting up long sentences.

lread committed:
Review and update README

As a noob, I did not at first understand Polylith was separate from the Clojure
poly tool and documented separately. I made this clearer in the README.
- changed the title to include "for Clojure"
- I used the term "conceptual" instead of "high-level" for Polylith
gitbook docs.

I got very confused by the diagram.
It is very cool but I thought it deserved at least a high level description.
I might have got it right, please do review.

Was not shy about using the word Polylith.
In a README, I felt it deserved to be repeated often.

Other changes were minor, like splitting up long sentences.

tengstrand commented on readme.adoc:
```diff
@@ -8,46 +8,52 @@ https://cljdoc.org/d/polylith/clj-poly/0.2.18-SNAPSHOT[image:https://badgen.net/
-The Polylith concept can be implemented in any programming language, but this version of the Polylith tool targets
-https://clojure.org/[Clojure] which is a powerful and simple functional language for the
-https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiB88eLxansAhUyi8MKHd6jDPEQFjAAegQIBRAC&url=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FJava_virtual_machine&usg=AOvVaw0YtnMyoG7GQIhUPeLulbfr[JVM].
+Polylith can be implemented in any programming language.
```
I suggest that we skip this sentence entirely.

lread replied:
Yeah, I think that make sense, it is now redundant.

lread committed:
Merge remote-tracking branch 'upstream/issue-318' into lread-readme-review

* upstream/issue-318:
  Moved Polylith to production-systems.adoc.

lread committed:
respond to review feedback

- turf redundant sentence
- turf diagram and description

lread committed:
Formatting only: switch to one sentence per line

As discussed on Slack.
https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

tengstrand commented on readme.adoc:
```diff
@@ -8,46 +8,36 @@ https://cljdoc.org/d/polylith/clj-poly/0.2.18-SNAPSHOT[image:https://badgen.net/
-They allow us to work with all our code in one place for maximum productivity, using a single
-https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop[REPL].
+The documentation you are reading now describes the Polylith tool built for https://clojure.org/[Clojure].
+Clojure is a powerful and simple functional language for the https://en.wikipedia.org/wiki/Java_virtual_machine[JVM].
```
I think it flows better if we merge these two sentences "...describes the Polylith tool built for [Clojure](https://clojure.org/), which is a powerful ...
To get rid of the repetition "Clojure. Clojure".

lread replied:
I've personally got no problem with the 2 Clojures. 
I think smaller sentences are easier to digest.

lread replied:
But, of course, your call.

lread replied:
Meh, I'll change it as suggested.

lread replied:
Done!

tengstrand replied:
Okay, thanks!

tengstrand commented on readme.adoc:
```diff
@@ -8,46 +8,36 @@ https://cljdoc.org/d/polylith/clj-poly/0.2.18-SNAPSHOT[image:https://badgen.net/
 
-Iron, Bronze, Silver, Gold, and Platinum sponsors will also be listed on the
-https://polylith.gitbook.io/polylith[first page] of the high-level documentation.
+Iron, Bronze, Silver, Gold, and Platinum sponsors are also listed and on the https://polylith.gitbook.io/polylith[front page of the Polylith docs].
```
Maybe it's enough to put the link in "front page": https://polylith.gitbook.io/polylith[front page] of the Polylith docs.

lread replied:
Sure, can do.

lread committed:
review feedback

reduce amount of text in link

lread committed:
review feedback - join 2 sentences

Compromised and got rid of a couple of unnecessary words.
